### PR TITLE
Adding clarifications on minimum required versions for out of band as…

### PR DESCRIPTION
**Pull request by e-shuster**
Adding clarifications on minimum required versions for out of band as well as a new section to troubleshooting helping to troubleshoot such issues

### CODE REVIEW

File: compute/admin_guide/waas/deploy_waas/deployment_out_of_band.adoc [hunk 1]
```diff
@@ -1,5 +1,7 @@
 == Deploy WAAS Out-Of-Band
 
+NOTE: A minimum 22.06 version of BOTH console and Defenders is required for Out-of-band.
```
Comment (divyabhushan-okta) [inline compute/admin_guide/waas/deploy_waas/deployment_out_of_band.adoc]: How about changing this as: 
"Out-Of-Band requires a minimum 22.06 version of both the Console and Defenders."

File: compute/admin_guide/waas/deploy_waas/deployment_vpc_mirroring.adoc [hunk 2]
```diff
@@ -1,5 +1,7 @@
 == Deploy WAAS Out-Of-Band with VPC Traffic Mirroring
 
+NOTE: A minimum 22.06 version of BOTH console and Defenders is required for VPC Traffic Mirroring.
```
Comment (divyabhushan-okta) [inline compute/admin_guide/waas/deploy_waas/deployment_vpc_mirroring.adoc]: VPC Traffic Mirroring requires a minimum 22.06 version of both the Console and Defenders.

File: compute/admin_guide/waas/out_of_band.adoc [hunk 3]
```diff
@@ -1,5 +1,7 @@
 == Deploying WAAS out of band
```
Comment (divyabhushan-okta) [inline compute/admin_guide/waas/out_of_band.adoc:2]: Isn't this file outdated? As we improved its content for OOB with VPC Traffic mirroring and moved it under "deploy_waas" directory. So, this file shouldn't be included, what do you suggest?

**Review by divyabhushan-okta — CHANGES_REQUESTED** [review]

File: compute/admin_guide/waas/deploy_waas/deployment_out_of_band.adoc (hunk 1 above)
Comment (e-shuster) [reply compute/admin_guide/waas/deploy_waas/deployment_out_of_band.adoc]: Sounds good

File: compute/admin_guide/waas/out_of_band.adoc (hunk 3 above)
Comment (e-shuster) [reply compute/admin_guide/waas/out_of_band.adoc:2]: Yes it is, I was not sure why it's still there so I added the comment just to be on the safe side. It's not part of the rendered docs

Comment (e-shuster) [reply compute/admin_guide/waas/out_of_band.adoc:2]: Let's make sure that other changes are properly published and i'll create a separate pull request to remove this file.

File: compute/admin_guide/waas/deploy_waas/deployment_out_of_band.adoc (hunk 1 above)
Comment (e-shuster) [reply compute/admin_guide/waas/deploy_waas/deployment_out_of_band.adoc]: @divyabhushan - The sentence has been changed as per your suggestion

File: compute/admin_guide/waas/deploy_waas/deployment_vpc_mirroring.adoc (hunk 2 above)
Comment (e-shuster) [reply compute/admin_guide/waas/deploy_waas/deployment_vpc_mirroring.adoc]: @divyabhushan - The sentence has been changed as per your suggestion

File: compute/admin_guide/waas/deploy_waas/deployment_out_of_band.adoc (hunk 1 above)
Comment (Pubs-MV) [reply compute/admin_guide/waas/deploy_waas/deployment_out_of_band.adoc]: @e-shuster , I moved it to the prereqs. Please take a look.

Comment (divyabhushan) [reply compute/admin_guide/waas/deploy_waas/deployment_out_of_band.adoc]: > 
@e-shuster Thank you, and yes this line would be better placed in Prerequisites section at line number 14.

File: compute/admin_guide/waas/deploy_waas/deployment_vpc_mirroring.adoc [hunk 4]
```diff
@@ -1,5 +1,7 @@
 == Deploy WAAS Out-Of-Band with VPC Traffic Mirroring
 
+NOTE: VPC Traffic Mirroring requires a minimum 22.06 version of both the Console and Defenders.
```
Comment (divyabhushan-okta) [inline compute/admin_guide/waas/deploy_waas/deployment_vpc_mirroring.adoc]: @e-shuster On this page, we refer to the agent as an Observer, so do you think we might need to edit the above sentence?

File: compute/admin_guide/waas/deploy_waas/deployment_out_of_band.adoc (hunk 1 above)
Comment (divyabhushan-okta) [reply compute/admin_guide/waas/deploy_waas/deployment_out_of_band.adoc]: @e-shuster This looks even better, thank you!

**Review by divyabhushan-okta — APPROVED** [review]

**Pull request merged**